Projekty/Projekt1/Fijalkowski_Krzyzinski_Nocon/EDA.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>Jest to dobry wykres, ale podobnie jak dla Work class przydałoby się pokazać (być może na oddzielnym wykresie) liczebność każdej z tych grup.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>